src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -1109,14 +1118,15 @@ public boolean tryToTakeFromListOrRequest(final boolean shouldRequest, final Ite
 
     /**
      * Check if a certain itemStack is already in the neededItems list.
+     *
      * @param tempStack the stack to test for.
      * @return true if so.
      */
     private boolean isInNeededItems(final ItemStack tempStack)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "isInNeededItems" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)